CorsixTH/Lua/dialogs/staff_dialog.lua

```diff
@@ -30,6 +30,13 @@ class "UIStaff" (Window)
 ---@type UIStaff
 local UIStaff = _G["UIStaff"]
 
+function UIStaff:afterLoad(old, new)
```

Comment:
I can happily move this for you, but we'll prefer this to float to the bottom of the file.